composer.json

```diff
@@ -53,11 +53,11 @@
       "@lint",
       "@sniff",
       "@security",
-      "@test"
+      "@coverage"
     ],
     "coverage": [
-      "git diff origin/master... -- > diff.txt",
-      "diffFilter --phpunit diff.txt build/logs/clover.xml 80"
+      "phpunit --dump-xdebug-filter=build/xdebug-filter.php",
```

Comment:
For reference, see https://phpunit.readthedocs.io/en/8.4/code-coverage-analysis.html#speeding-up-code-coverage-with-xdebug.